packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -1485,10 +1407,14 @@ describe('Dropdown', () => {
       const stopImmediatePropagation = jest.fn()
       const mockedEvent = { stopPropagation, nativeEvent: { stopImmediatePropagation } }
```

Comment:
I don't think that we need `mockedEvent` anymore as `clickOnSelectedItemAtIndex()` handles this

Reply:
awesome, thanks!